lib/consyncful.rb

```diff
@@ -22,23 +20,39 @@ class Configuration
                   :preserve_contentful_timestamps
 
     def initialize
-      @contentful_client_options = {
-        api_url: 'cdn.contentful.com'
```

Comment:
I removed this because it looks like we'll get it from the DEFAULT_CLIENT_OPTIONS anyway